### PR TITLE
Add --[no-]post-install-message argument to gem install/update.

### PR DESCRIPTION
**Pull request by simi**
Hello!

Often I don't need to read post install messages (and they are really long some times). I've added option to skip them in install/update process.

It is really naive implementation.

Feel free to ping me if this change makes sense at all and if some improvements are needed.

*idea by https://github.com/tpope/gem-shut-the-fuck-up*

### CODE REVIEW

**Comment by grosser** [comment]
oh yeah! :+1:

File: test/rubygems/test_gem_install_update_options.rb [hunk 1]
```diff
@@ -181,4 +182,15 @@ def test_vendor_missing
     RbConfig::CONFIG['vendordir'] = orig_vendordir
   end
 
+  def test_post_install_message_no
```
Comment (grosser) [inline test/rubygems/test_gem_install_update_options.rb]: maybe also test that it is true by default

Comment (simi) [inline test/rubygems/test_gem_install_update_options.rb]: That can't be tested here. But I can test it in installer.

**Comment by simi** [comment]
I think failing test is note related to this PR.

**Comment by simi** [comment]
I reopened this to get CI triggered and it is :green_heart: now.

**Comment by swills** [comment]
:thumbsup:

**Comment by indirect** [comment]
@simi can you rebase and green up the build? I'd be happy to merge this.

**Comment by simi** [comment]
@indirect done!

**Comment by indirect** [comment]
Thanks!

**Pull request merged**